activiti-cloud-services-runtime-bundle/activiti-cloud-services-rest-impl/src/test/java/org/activiti/cloud/services/rest/controllers/TaskControllerImplIT.java

```diff
@@ -209,6 +209,16 @@ public void completeTask() throws Exception {
                                 pathParameters(parameterWithName("taskId").description("The task id"))));
     }
 
+    @Test
+    public void saveTask() throws Exception {
+        this.mockMvc.perform(post("/v1/tasks/{taskId}/save",
```

Comment:
We should provide a payload with variables here I think.

Reply:
@erdemedeiros I wast looking into post(..) signature but there is no body argument provided.

Reply:
@erdemedeiros Ok, I found how to set the content body. See: https://github.com/Activiti/activiti-cloud-runtime-bundle-service/pull/296/commits/66433f0c7483210dcdf0f1ab53de88be1f8a0545